industry_benchmarks/utils/data_gathering.py

```diff
@@ -496,6 +540,18 @@ def gather_data(
         fixed_network = parse_ligand_network(fixed_ligand_network)
         ligand_network = ligand_network.enlarge_graph(
             edges=fixed_network.edges, nodes=fixed_network.nodes)
+
+    if hide_ligand_names:
+        ligand_network, name_mapping = replace_ligand_names(ligand_network)
+        with open(output_dir / "ligand_name_mapping.json", "w") as out:
+            json.dump(name_mapping, out)
+    else:
+        # make some noise that we are not hiding the names
+        import warnings
+        warnings.warn(message="The names of the ligands will be used in the results, if you want to make them anonymous "
```

Comment:
```suggestion
        warnings.warn(message="The names of the ligands will be used as-is in the results, if you want to make them anonymous "
```
Or something like that - the wording just seemed a tiny bit unclear sorry.